coderdojochi/admin.py

```diff
@@ -109,7 +114,41 @@ def get_last_name(self, obj):
     get_last_name.admin_order_field = 'user__last_name'
 
 
-class GuardianAdmin(admin.ModelAdmin):
+class GuardianImportResource(ModelResource):
+    first_name = Field(attribute='first_name', column_name='first_name')
+    last_name = Field(attribute='last_name', column_name='last_name')
+    email = Field(attribute='email', column_name='email')
+    phone = Field(attribute='phone', column_name='phone')
+    zip = Field(attribute='zip', column_name='zip')
+
+    def get_or_init_instance(self, instance_loader, row):
+        return Guardian(user=User()), True
```

Comment:
@ash6851 Should this always create a new guardian? What if the guardian exists (e.g. accidentally importing someone who already exists)?